docker-compose.yml

```diff
@@ -0,0 +1,47 @@
+---
+version: '3'
+services:
+  db:
+    image: postgres:11.5-alpine
+    environment:
+      POSTGRES_DB: 'mydb'
+      POSTGRES_PASSWORD: 'hunter2'
+      POSTGRES_USER: 'roto_rooter'
```

Comment:
Shouldn't we go for a bit more Blossom-like credentials?

Reply:
Probably, but that was to test that we're actually pulling in values from environment variables and showing you could make the creds _anything_. And that's important from a security standpoint if we need to quickly pivot.

Here's an example: If username/password get owned, rotating just the password is going to leave it open to brute force attack. They know the first factor (username) and they can repeatedly guess at the second (password). Eventually they'll guess right. Will we notice before that happens and lock down access? Hopefully. I'd rather not make that wager in the first place.

Reply:
That makes sense, my remark was indeed about the default values in code :)